tests/e2e/env/setup.sh

```diff
@@ -75,8 +92,10 @@ if [[ "$E2E_USE_LOCAL_SERVER" != false ]]; then
 
 	if [[ -n $CI ]]; then
 		echo "Setting docker folder permissions"
-		redirect_output sudo chown www-data:www-data -R ./docker/wordpress
-		redirect_output ls -al ./docker
+		handle_permissions "$SERVER_PATH/docker/wordpress"
+		redirect_output ls -al "$SERVER_PATH/docker"
```

Comment:
We can remove this. It was added to debug file permission issues and forgot to remove it.

Reply:
Done in 6c73e5f04